jdbc/src/java/org/apache/hive/jdbc/Utils.java

```diff
@@ -105,6 +105,8 @@
     public static final String USE_SSL = "ssl";
     public static final String SSL_TRUST_STORE = "sslTrustStore";
     public static final String SSL_TRUST_STORE_PASSWORD = "trustStorePassword";
+    public static final String SSL_TRUST_STORE_TYPE = "trustStoreType";
```

Comment:
So now instead of defaulting to JKS, the client always has to provide "trustStoreType=JKS" in the connection params? Wouldnt this break backward compatibility with existing applications ?

Reply:
No, this param is optional, if it does not have value, KeyStore.getDefaultType() will be used.